### PR TITLE
Improvement/import from seed styles

### PR DESCRIPTION
**Pull request by estebanmino**
**Description**

Updates Import from Seed onboarding flow to match the rest of onboarding flow

image

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/components/Views/ImportFromSeed/index.js [hunk 1]
```diff
@@ -489,9 +428,13 @@ class ImportFromSeed extends PureComponent {
 							value={this.state.seed}
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:428]: `this.state.seed` --> `seed`

File: app/components/Views/ImportFromSeed/index.js [hunk 2]
```diff
@@ -489,9 +428,13 @@ class ImportFromSeed extends PureComponent {
...
 							numberOfLines={3}
 							multiline
-							style={[styles.seedPhrase, this.state.inputWidth ? { width: this.state.inputWidth } : {}]}
+							style={[
+								styles.seedPhrase,
+								this.state.inputWidth ? { width: this.state.inputWidth } : {},
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:433]: Use binary op `inputWidth && { width: this.state.inputWidth }`

File: app/components/Views/ImportFromSeed/index.js [hunk 3]
```diff
@@ -489,9 +428,13 @@ class ImportFromSeed extends PureComponent {
...
+								this.state.seedphraseInputFocused && styles.inputFocused
+							]}
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:435]: `seedphraseInputFocused`

File: app/components/Views/ImportFromSeed/index.js [hunk 4]
```diff
@@ -500,46 +443,35 @@ class ImportFromSeed extends PureComponent {
...
-								secureTextEntry={this.state.secureTextEntry}
-								placeholder={''}
-								placeholderTextColor={colors.grey100}
-								underlineColorAndroid={colors.grey100}
-								onSubmitEditing={this.jumpToConfirmPassword}
 								returnKeyType={'next'}
-								onFocus={() => this.animateOutLabel('new')} // eslint-disable-line  react/jsx-no-bind
-								onBlur={() => this.animateInLabel('new')} // eslint-disable-line  react/jsx-no-bind
 								autoCapitalize="none"
+								secureTextEntry={this.state.secureTextEntry}
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:460]: `secureTextEntry `

File: app/components/Views/ImportFromSeed/index.js [hunk 5]
```diff
@@ -500,46 +443,35 @@ class ImportFromSeed extends PureComponent {
...
+								onChangeText={this.onPasswordChange}
+								value={this.state.password}
+								baseColor={colors.grey500}
+								tintColor={colors.blue}
+								onSubmitEditing={this.jumpToConfirmPassword}
+								renderRightAccessory={() => (
+									<TouchableOpacity onPress={this.toggleShowHide} style={styles.showHideToggle}>
+										<Text style={styles.passwordStrengthLabel}>
+											{strings(`choose_password.${this.state.secureTextEntry ? 'show' : 'hide'}`)}
+										</Text>
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:470]: `secureTextEntry`

File: app/components/Views/ImportFromSeed/index.js [hunk 6]
```diff
@@ -552,36 +484,21 @@ class ImportFromSeed extends PureComponent {
...
+								onChangeText={this.onPasswordConfirmChange}
+								returnKeyType={'next'}
+								autoCapitalize="none"
 								secureTextEntry={this.state.secureTextEntry}
-								placeholder={''}
-								placeholderTextColor={colors.grey100}
-								underlineColorAndroid={colors.grey100}
+								placeholder={strings('import_from_seed.confirm_password')}
+								value={this.state.confirmPassword}
+								baseColor={colors.grey500}
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:497]: `confirmPassword `

File: app/components/Views/ImportFromSeed/index.js [hunk 7]
```diff
@@ -552,36 +484,21 @@ class ImportFromSeed extends PureComponent {
...
-								value={this.state.confirmPassword}
-								onChangeText={this.onPasswordConfirmChange} // eslint-disable-line  react/jsx-no-bind
+								onChangeText={this.onPasswordConfirmChange}
+								returnKeyType={'next'}
+								autoCapitalize="none"
 								secureTextEntry={this.state.secureTextEntry}
-								placeholder={''}
-								placeholderTextColor={colors.grey100}
-								underlineColorAndroid={colors.grey100}
+								placeholder={strings('import_from_seed.confirm_password')}
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:495]: `secureTextEntry `

File: app/components/Views/ImportFromSeed/index.js [hunk 8]
```diff
@@ -500,46 +443,35 @@ class ImportFromSeed extends PureComponent {
...
 								autoCapitalize="none"
+								secureTextEntry={this.state.secureTextEntry}
+								onChangeText={this.onPasswordChange}
+								value={this.state.password}
+								baseColor={colors.grey500}
+								tintColor={colors.blue}
+								onSubmitEditing={this.jumpToConfirmPassword}
+								renderRightAccessory={() => (
+									<TouchableOpacity onPress={this.toggleShowHide} style={styles.showHideToggle}>
+										<Text style={styles.passwordStrengthLabel}>
```
Comment (EtDu) [inline app/components/Views/ImportFromSeed/index.js:468]: perhaps change the naming: `toggleShowHide` --> `toggleSecureTextEntry` for clarity

**Review by EtDu — CHANGES_REQUESTED** [review]
Looks good, some minor fixes

File: app/components/Views/ImportFromSeed/index.js [hunk 9]
```diff
@@ -451,33 +412,11 @@ class ImportFromSeed extends PureComponent {
...
-
-		const labelsScaleNewX = this.state.labelsScaleNew.interpolate({
-			inputRange: [0, 1],
-			outputRange: [startX - width / 2 - (width * initialScale) / 2, endX]
-		});
-		const labelsScaleNewY = this.state.labelsScaleNew.interpolate({
-			inputRange: [0, 1],
-			outputRange: [startY - height / 2 - (height * initialScale) / 2, endY]
-		});
+	setSeedphraseInputFocused = () => this.setState({ seedphraseInputFocused: true });
```
[the remark this answers is not in the document]
Comment (estebanmino) [reply app/components/Views/ImportFromSeed/index.js:415]: I need to explicitly have that, those actions are going to be triggered when the app state is different than active, so I could be prompting the modal without any transaction if I do that

**Review by andrepimenta — APPROVED** [review]
LGTM! 👍

**Review by ibrahimtaveras00 — APPROVED** [review]
QA Passed 👍

**Review by EtDu — APPROVED** [review]
LGTM

**Pull request merged**